apps/src/code-studio/header.js

```diff
@@ -146,6 +157,12 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
   }
 };
 
+const ConnectedShareDialog = connect(state => ({
+  isOpen: state.header.shareDialog.isOpen,
+}), dispatch => ({
+  onClose: () => dispatch(hideShareDialog())
+}))(ShareDialog);
```

Comment:
I would prefer any `connect` to be the the same file as the component itself. So either ShareDialog can export both a connected and an unconnected version, or we have a separate file for ConnectedShareDialog (I would probably lean towards former).